js/libs/core.js

```diff
@@ -489,10 +489,76 @@ Fliplet.FormBuilder = (function() {
         return !_.isEmpty(this.errors);
       };
 
+      component.methods._hasDuplicateOptions  = function(options) {
+        var finalOptions = _.cloneDeep(options);
+
+        _.forEach(finalOptions, function(i) {
+          i.id = i.id ? i.id.toLowerCase() : i.label.toLowerCase();
+          i.label = i.label ? i.label.toLowerCase() : i.id.toLowerCase();
+
+          return i;
+        });
+
+        var duplicates = _.filter(
+          _.uniq(
+            _.map(finalOptions, function(item) {
+              var val = (item.id ? item.id : item.label).toLowerCase();
```

Comment:
Why do you still need to check for item.id when you've mapped them all above to always have both `id` and `label`?

Reply:
Yes, agreed, directly using item.id.toLoweCase().